docs/how-to/peering-with-content-providers.md

```diff
@@ -60,32 +60,19 @@ This list is provided for informational purposes only. The IPFS Project does not
 | `Qmcfuo1TM9uUiJp6dTbm915Rf1aTqm3a3dnmCdDQLHgvL5` | `/dnsaddr/node-11.ingress.cloudflare-ipfs.com` |
 | `QmcfV2sg9zaq7UUHVCGuSvT2M2rnLBAPsiE79vVyK3Cuev` | `/dnsaddr/node-12.ingress.cloudflare-ipfs.com` |
 
-### European EPC Competence Center
+### Filebase
 
 | Peer ID | Addresses |
 | ------- | --------- |
-|`12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME`|`/dnsaddr/ipfs.ssi.eecc.de`|
```

Comment:
I was able to connect to this node and ping it. Was there anything specific that failed with this node?

Reply:
```
$ vole libp2p identify /dnsaddr/ipfs.ssi.eecc.de/tcp/4001/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME
panic: failed to dial: failed to dial 12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME: no good addresses
```

Reply:
Weird:

```
➜  ~ ipfs ping 12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME
Looking up peer 12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME
PING 12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME.
Pong received: time=32.41 ms
Pong received: time=719.12 ms
^C
Average latency: 375.76ms
➜  ~ ipfs id 12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME
{
	"ID": "12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME",
	"PublicKey": "CAESIGRmg+kLxRhZT4VzV9FC+CZ3IKMjaTv3ZBsV68nrQMCF",
	"Addresses": [
		"/ip4/49.12.122.178/tcp/1024/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME",
		"/ip4/49.12.122.178/tcp/4001/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME",
		"/ip4/49.12.122.178/udp/4001/quic-v1/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME",
		"/ip4/49.12.122.178/udp/4001/quic-v1/webtransport/certhash/uEiBvHPJQQ5LUKg9pESpzVXjlu6uFwYoHVtR51owoi4bpfg/certhash/uEiClITdvqcKsjfaI-jfoFGXXWOlN9Rnc9n0_sXqhGFXqVg/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME"
	],
	"AgentVersion": "kubo/0.27.0/59bcea8/docker",
	"Protocols": [
		"/ipfs/bitswap",
		"/ipfs/bitswap/1.0.0",
		"/ipfs/bitswap/1.1.0",
		"/ipfs/bitswap/1.2.0",
		"/ipfs/id/1.0.0",
		"/ipfs/id/push/1.0.0",
		"/ipfs/kad/1.0.0",
		"/ipfs/lan/kad/1.0.0",
		"/ipfs/ping/1.0.0",
		"/libp2p/autonat/1.0.0",
		"/libp2p/circuit/relay/0.2.0/hop",
		"/libp2p/circuit/relay/0.2.0/stop",
		"/libp2p/dcutr",
		"/x/"
	]
}

➜  ~ dig ipfs.ssi.eecc.de

; <<>> DiG 9.10.6 <<>> ipfs.ssi.eecc.de
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 61682
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;ipfs.ssi.eecc.de.		IN	A

;; ANSWER SECTION:
ipfs.ssi.eecc.de.	3276	IN	A	49.12.122.178

;; Query time: 66 msec
;; SERVER: 192.168.3.1#53(192.168.3.1)
;; WHEN: Wed Apr 17 18:32:37 CEST 2024
;; MSG SIZE  rcvd: 61

➜  ~
```

Reply:
I think the problem is with Vole which isn't able to resolve `/dnsaddr/` multiaddrs.

I was able to run identify with the ip multiaddr:
```
vole libp2p identify /ip4/49.12.122.178/tcp/4001/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME
PeerID: "12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME"
Protocol version: ""
Agent version: "kubo/0.27.0/59bcea8/docker"
Listen addresses:
	- "/ip4/49.12.122.178/tcp/1024/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME"
	- "/ip4/49.12.122.178/tcp/4001/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME"
	- "/ip4/49.12.122.178/udp/4001/quic-v1/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME"
	- "/ip4/49.12.122.178/udp/4001/quic-v1/webtransport/certhash/uEiBvHPJQQ5LUKg9pESpzVXjlu6uFwYoHVtR51owoi4bpfg/certhash/uEiClITdvqcKsjfaI-jfoFGXXWOlN9Rnc9n0_sXqhGFXqVg/p2p/12D3KooWGaHbxpDWn4JVYud899Wcpa4iHPa3AMYydfxQDb3MhDME"
Protocols:
	- "/ipfs/bitswap"
	- "/ipfs/bitswap/1.0.0"
	- "/ipfs/bitswap/1.1.0"
	- "/ipfs/bitswap/1.2.0"
	- "/ipfs/id/1.0.0"
	- "/ipfs/id/push/1.0.0"
	- "/ipfs/kad/1.0.0"
	- "/ipfs/lan/kad/1.0.0"
	- "/ipfs/ping/1.0.0"
	- "/libp2p/autonat/1.0.0"
	- "/libp2p/circuit/relay/0.2.0/hop"
	- "/libp2p/circuit/relay/0.2.0/stop"
	- "/libp2p/dcutr"
	- "/x/"
```

Reply:
@2color `vole` is able to resolve `/dnsaddr` multiaddrs. The problem here is that the DNS record in question actually doesn't have a `_dnsaddr` TXT record defined - thus `/dnsaddr` will not work.

I've gone ahead and add the peer ID back, but with an updated `/dns4` prefix.